Alert/doc/spec.md

```diff
@@ -0,0 +1,148 @@
+# Alert data model
+
+## Description
+This entity models an alert generated either by a user or by a device. This models could be used to send alerts related to traffic jam, accidents, weather conditions, high level of pollutants and so on.
+The purpose of the model is to support the generation of notifications for a user or trigger other actions,
+based on such alerts.
+
+### Alert definition
+Alerts are context data generated by final users (humans) or devices, such as smartphones, service, or sensors.
+
+An alert is generated by a specific situation. The main features of an alert is that it is not predictable and it is not a recurrent data. That means that an alert could be an accident or a high level of pollutants measure, additionally it could be the fall down of a patient or a car driving in the opposite direction.
+
+The model has been defined to support different scenarios inside [SmartSDK](https://smartsdk.eu) project:
+
+| Smart city  | Smart security | Smart health |
+| ------------- | ------------- |------------- |
+| Traffic jam  | A car in opposite direction  |Fall of a patient |
+| Accident cars  | When a car suddenly stops  |Bump of a patient |
+| Weather condition  | Suspicious activities  | |
+| High level of pollution  | Assaults | |
+| Pollen | | |
+| Asthma attacks || |
+
+It is important to remark that an alert can be generated by a user or device. Currently, for smart city scenario a set of alerts have been detected, such as: traffic jam, accident cars, weather condition, pollution, pollen and asthma attacks. In addition, for each type of alert, also a set of sub-types of alerts have been identified. For instances, subtypes of alerts of traffic jam: heavy traffic jam, car stopped on road, and so on. A catalogue of alerts subtypes for smart city scenario have been defined and it is showed to below.
+
+#### Alert catalogue
+
+| Traffic| Accidents | Weather conditions | Pollutions | Pollen |Asthma attacks |
+| ------------- | ------------- |------------- | ------------- | ------------- |------------- |
+| Standstill Traffic Jam |Minor Accident|Rainfall|Visible pollution (smog)|Symptoms or discomforts of users| People with asthma attacks at specific point. |
+| Heavy Traffic Jam |Car Accident|Temperature (low, high)|Level of emissions|Level of pollen on air| |
+| Bumper to bumper traffic|Hazard On Road|Heat alert|Level of pollutants, such as O3.|Pollen concentration| |
+| Moderate Traffic Jam|Assaults|Wind|Pre-contingency alert| | |
+| Car Stopped On Road|Bikers injured|Flood warning|Contingency alert| | |
+| Roadworks|Car crashes|dense fog|| | |
+| Road closed|Suspicious activities|Tornado | | | |
+| Pothole| |Tropical cyclone| | | |
+| Congestion on road| |Hurricane| | | |
+| Car in opposite way| |Tsunami| | | |
+| When a car suddenly stops | | | | | |
+
```

Comment:
This should be moved to the place where the different enumeration values are described. Clear enumeration values have to be provided following the camel case conventions as it happens with other data models.